test/test-utils.js

```diff
@@ -43,3 +44,23 @@ test('joinListConjuction test', function (t) {
   t.equal(utils.joinListConjuction(things, ', ', ' and '), 'this, that and other');
   t.equal(utils.joinListConjuction(thing, ', ', ' and '), 'this');
 });
+
+test('makeGetStationResponseText tests', function (t) {
+  t.plan(3);
+  var multipleStationArrivals = {
+    'largo town center': ['4', '8'],
+    'vienna': ['12', '16'],
+    'wiehle reston east': ['9'],
+    'new carrollton': ['2']
+  };
+  var oneStationArrival = {
+    'vienna': ['7']
+  };
+  var stationName = 'ballston';
+
+  t.equal(utils.makeGetStationResponseText(multipleStationArrivals, stationName), 'Are you going to largo town ' +
+    'center, vienna, wiehle reston east or new carrollton');
+  t.equal(utils.makeGetStationResponseText(oneStationArrival, stationName), 'The next train to vienna arrives in 7 ' +
+    'minutes. There are no other trains arriving at ballston in the next 20 minutes.');
+  t.equal(utils.makeGetStationResponseText({}, stationName), props.stationNoArrivalsSpeechOutput);
```

Comment:
Let's hard code the expected response instead of loading from properties. That way, the test breaks if the property is changed. Then, we have to consciously choose to fix the test if it is the correct change.